app-kotlin/src/main/res/layout/error_view.xml

```diff
@@ -1,11 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ImageView xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/image_error"
-    android:layout_width="100dp"
-    android:layout_height="100dp"
-    android:layout_gravity="center"
-    android:visibility="gone"
-    app:contentDescription="error image"
-    app:srcCompat="@drawable/ic_error_round" />
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
 
+    <ProgressBar
+        android:id="@+id/home_progressBar"
```

Comment:
this is reused by both `home` and `details` right? 🤔 calling this home makes less sense now

Reply:
Good point I had changed it to progressBar